easybuild/easyconfigs/s/Seurat/Seurat-4.1.0-foss-2021a-R-4.1.0.eb

```diff
@@ -0,0 +1,75 @@
+# #
+# This is a contribution from HPCNow! (http://hpcnow.com)
+# Copyright::   HPCNow!
+# Authors::     Arturo Gimeno <arturo.gimeno@hpcnow.com>, Pau Ruiz
+# License::     GPL-v3.0
+# #
+
+easyblock = 'Bundle'
+
+name = 'Seurat'
+version = '4.1.0'
+versionsuffix = '-R-%(rver)s'
+
+homepage = 'https://satijalab.org/seurat'
+description = "Seurat is an R package designed for QC, analysis, and exploration of single cell RNA-seq data."
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+dependencies = [
+    ('R', '4.1.0'),
+    ('R-bundle-Bioconductor', '3.13', versionsuffix),
+]
+
+exts_defaultclass = 'RPackage'
+
+exts_default_options = {
+    'source_urls': [
+        'https://cran.r-project.org/src/contrib/Archive/%(name)s',  # package archive
+        'https://cran.r-project.org/src/contrib/',  # current version of packages
+        'https://cran.freestatistics.org/src/contrib',  # mirror alternative for current packages
+    ],
+    'source_tmpl': '%(name)s_%(version)s.tar.gz',
+}
+
+exts_list = [
+    ('Matrix', '1.5-1', {
+        'checksums': ['557dba0358172d67dc63eb5db90841915bb5ce1528f941a8005ae808d635575d'],
+    }),
+    ('scattermore', '0.7', {
+        'checksums': ['f36280197b8476314d6ce81a51c4ae737180b180204043d2937bc25bf3a5dfa2'],
+    }),
+    ('rlang', '0.4.10', {
+        'checksums': ['07530270c4c199f2b7efc5d57a476d99babd9d0c3388a02bb7d57fe312da3576'],
+    }),
+    ('RcppAnnoy', '0.0.18', {
+        'checksums': ['e4e7ddf071109b47b4fdf285db6d2155618ed73da829c30d8e64fc778e63c858'],
+    }),
+    ('Rcpp', '1.0.7', {
+        'checksums': ['15e5a4732216daed16263c79fb37017c2ada84a2d4e785e3b76445d0eba3dc1d'],
+    }),
+    ('rgeos', '0.6-3', {
+        'checksums': ['1ebc5a99fdcf7d293811ce26379ba1525a2b9f23b11f1fb4a9c582bf27fc2b3c'],
+    }),
+    ('SeuratObject', version, {
+        'checksums': ['9ca406cb3bd95c588e1a81c5383e3173a446cc0667142b139ca32685b4b20a05'],
+    }),
+    ('sctransform', '0.3.5', {
+        'checksums': ['c08e56df05d64ed04ee53eb9e1d4d321da8aff945e36d56db1d5ceb1cd7e6e0b'],
+    }),
+    ('uwot', '0.1.10', {
+        'checksums': ['6ee1b6027bce679cd5a35f647f516a5b327632234bcf323c7f3d5b5e10807d23'],
+    }),
+    (name, version, {
+        'checksums': ['2505829a2763e449684dd1b107ee6982e019ee9fecb093adca7b283cad1b315d'],
+    }),
+]
+
+modextrapaths = {'R_LIBS': ''}
```

Comment:
@agimenog please note that the unit tests are complaining about this `R_LIBS`, which is supposed to be `R_LIBS_SITE`. Is there a reason you have that value here?

Reply:
hi @verdurin, I just saw the logs and you're right, I changed it